src/components/templates/Header.jsx

```diff
@@ -53,12 +53,12 @@ const Header = () => {
           <div
             className={
               dropDown
-                ? "delay-5000 flex flex-col gap-y-3 bg-white transition-transform duration-300 ease-in-out"
-                : "delay-5000 h-0 w-0 overflow-hidden transition-transform duration-300 ease-in-out lg:ml-auto lg:flex lg:h-auto lg:w-max lg:gap-x-8 xl:gap-x-16"
+                ? "z-10 delay-5000 absolute left-0 right-0 top-14 flex w-full flex-col gap-y-3 rounded-lg bg-white transition-transform duration-300 ease-in-out"
+                : "delay-5000 h-0 w-0 overflow-hidden transition-transform duration-300 ease-in-out lg:ml-auto lg:flex lg:h-auto lg:w-max  xl:gap-x-16"
             }
           >
             <nav className="text-sm lg:flex lg:items-center">
-              <ul className="flex flex-col lg:flex-row lg:items-center">
+              <ul className="flex flex-col pl-4 lg:flex-row lg:items-center">
                 <li className="px-2 py-2  lg:px-1.5">
```

Comment:
Create an array of objects for links and map them, don't hard code and duplicate links.